zstacklib/zstacklib/utils/file_metadata_handler.py

```
@@ -0,0 +1,233 @@
+                    summary_path, vmUuid,
+                    vm_name=vmName,
+                    vm_category=vmCategory,
+                    architecture=architecture,
+                    schema_version=schemaVersion,
+                )
+            else:
+                # No vmUuid: remove stale summary to avoid scan() returning outdated info
+                try:
+                    if os.path.exists(summary_path):
+                        os.remove(summary_path)
+                except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)